How-to-upgrade-review-version.md

```diff
@@ -0,0 +1,399 @@
+# How to Upgrade to TypeScript Without Anybody Noticing
+
+This guide will show you how to upgrade to Typescript without anybody
+noticing. Well, people *might* notice &mdash; what I really mean is
+that you won't have to change your build at all. You'll have the
```

Comment:
Since the premise of the article is to be sneaky, perhaps summarize what I end up having to check into source control that are TypeScript-specific. As far as I can tell, that's a `tsconfig.json` and some new `@types` dev dependencies in `package.json`.